### ISSUE TITLE
`integrity` attributes with multiple hashes are wrongly flagged as warnings

### ISSUE THREAD

**Issue by cvrebert** [validator/validator#212]
Example: https://validator.w3.org/nu/?doc=http%3A%2F%2Fbrowserbu.gs%2Fcss-hacks%2F

> Warning: Potentially bad value sha256-7s5uDGW3AHqw6xtJmNNtr+OBRJUlgkNJEo78P4b0yRw= sha512-nNo+yCHEyn0smMxSswnf/OnX6/KwJuZTlNZBjauKhTK0c+zT+q5JOCx0UFhXQ6rJR9jg6Es8gPuD2uZcYDLqSw== for attribute integrity on element link: Invalid base64-value (bad padding). Consider using RFC4648 compliant base64 encoding implementation

The markup in question:
```html
<link
  href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css"
  rel="stylesheet" 
  integrity="sha256-7s5uDGW3AHqw6xtJmNNtr+OBRJUlgkNJEo78P4b0yRw= sha512-nNo+yCHEyn0smMxSswnf/OnX6/KwJuZTlNZBjauKhTK0c+zT+q5JOCx0UFhXQ6rJR9jg6Es8gPuD2uZcYDLqSw=="
  crossorigin="anonymous"
/>
```

Per [the grammar in the SRI spec](http://www.w3.org/TR/SRI/#the-integrity-attribute), the `integrity` attribute can have multiple whitespace-separated hashes.

(Or perhaps there's some other syntax error afoot here that I need to let BootstrapCDN know about?)

**Comment by sideshowbarker** [comment]
> Per the grammar in the SRI spec, the integrity attribute can have multiple whitespace-separated hashes.
> 
> (Or perhaps there's some other syntax error afoot here that I need to let BootstrapCDN know about?)

I think there’s no other syntax error—instead it’s just that I mistakenly only implemented support for single values. So I just need to add a splitter to the datatype-checking code for it, and will get that done soon, and get a new bug-fix release out (I’ve made one other bug fix so far, for issue #208, that needs to be released, along with needing to merge PR #210).
